src/api/apps/users/model.coffee

```diff
@@ -25,16 +25,18 @@ jwtDecode = require 'jwt-decode'
     return callback err if err
     db.users.findOne { access_token: encryptedAccessToken }, (err, user) ->
       return callback err if err
-      return callback null, user if user
+      if user
+        savedPartnerIds = user.partner_ids.map((a) => a.toString())
```

Comment:
A comment here on the difference between the saved and latest might be helpful for future context.